policy/scan/local_scanner_test.go

```diff
@@ -138,10 +138,10 @@ func (s *LocalScannerSuite) BeforeTest(suiteName, testName string) {
 							{
 								Type: "k8s",
 								Options: map[string]string{
-									"path": "./testdata/2pods.yaml",
+									"path": "./testdata/1pod.yaml",
 								},
 								Discover: &inventory.Discovery{
-									Targets: []string{"auto"},
+									Targets: []string{"pods"},
```

Comment:
Why needs this test to change?

Reply:
I wanted to have 1 report instead of 3. Easier to write the tests